Jenkinsfile

```diff
@@ -15,6 +15,11 @@ node {
         ]),
     ]);
 
+    if ("${BRANCH_NAME}" =~ "^publish-.*") {
+        currentBuild.result = 'SUCCESS'
+        return
```

Comment:
Can haz semicolons?

Reply:
https://dzone.com/articles/groovy-sometimes-you-still

Groovy has fewer edge cases, but I will never remember them either.